nvflare/private/fed/server/job_runner.py

```diff
@@ -385,6 +385,9 @@ def run(self, fl_ctx: FLContext):
                     )
 
                     if ready_job:
+                        reload_job = job_manager.get_job(ready_job.job_id, fl_ctx)
+                        if reload_job.meta.get(JobMetaKey.STATUS) != RunStatus.SUBMITTED:
```

Comment:
What do you think if we wrap this two lines as a simple method like
`_check_job_status(job_manager, job_id, job_run_status)`
or add it in job_manager class.

And then in here we just call `if _check_job_status(job_manager, ready_job.job_id, RunStatus.SUBMITTED)`

Reply:
yeah, that's a little cleaner. changed it.